Makefile

```diff
@@ -11,7 +11,7 @@ build.all:
 	@rm -rf capten
 	@mkdir capten
 	@go mod download && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o capten/capten cmd/main.go
-#	@go mod download && GOOS=darwin GOARCH=amd64 go build -o capten/capten.app cmd/main.go
```

Comment:
Make should have separate build for mac and linux, it will generate 2 separate zip files for mac and linux
build.all: build.linux build.mac
build.linux:
-- ---- steps build zip file for linux
build.mac:
-- ---- steps build zip file for mac